src/main/webapp/app/entities/categorical-alteration/categorical-alteration.tsx

```diff
@@ -17,9 +20,29 @@ export const CategoricalAlteration = (props: ICategoricalAlterationProps) => {
   }, []);
 
   const columns: SearchColumn<ICategoricalAlteration>[] = [
-    { accessor: 'alterationType', Header: 'Alteration Type' },
-    { accessor: 'type', Header: 'Type' },
-    { accessor: 'name', Header: 'Name' },
+    {
+      accessor: 'alterationType',
+      Header: 'Alteration Type',
+      onFilter: (data: ICategoricalAlteration, keyword) => (data.alterationType ? filterByKeyword(data.alterationType, keyword) : false),
+    },
+    {
+      accessor: 'type',
+      Header: 'Type',
+      onFilter: (data: ICategoricalAlteration, keyword) => (data.type ? filterByKeyword(data.type, keyword) : false),
+    },
+    {
+      accessor: 'name',
+      Header: 'Name',
+      onFilter: (data: ICategoricalAlteration, keyword) => (data.name ? filterByKeyword(data.name, keyword) : false),
+    },
+    {
+      id: 'consequence',
```

Comment:
We're using the sync OncoKbTable here, so we should add `showPagination` as prop.
Currently can't see pages
![image](https://github.com/oncokb/oncokb-transcript/assets/59149377/032417cf-5f95-45f7-b1ae-2d4aab466a17)

Sorting not working because no accessor to get the term to compare
```suggestion
      accessor: 'consequence.term',
```